Fix for: add product showing error #21

andrevanzuydam opened the pull request:
add product showing error #21

andrevanzuydam committed:
Fix for:
add product showing error #21

andrevanzuydam commented:
More fixes coming, I'm doing each one as a separate pull request to try and align with reported issues.

andrevanzuydam commented:
Fixes Error setup:di:compile #19 this issue as well

andrevanzuydam commented:
Fixes Any up-to-date Multi-Vendor Marketplaces? #10 as well

andrevanzuydam commented:
Fixes Compilation Issue in Php7.2.16 environment #9 as well

andrevanzuydam commented:
Fixes Cannot add configurable product #7 as well

andrevanzuydam committed:
Fix for: Error when uploading product photo #15

godilite commented:
Please can someone help me, I have installed this extension on magneto 2.3.x, everything works fine except that the seller can't find a form to post products, the manage product page on the seller dashboard shows the title *Account* and nothing else except a white page. Please I really need help from everyone

godilite commented:
@andrevanzuydam please is there something I am not doing right? During installation? There is no guide on how to configure this extension after installation.

andrevanzuydam commented:
@godilite Hey there sorry for the late reply, did you test with the changes I've pushed already, there are still more errors to fix on the source but I am able to add products now but not attributes?

godilite commented:
I have not been able to fix it. Please can you tell me step by step how to install and setup this extension on magneto2?

andrevanzuydam commented:
Hi

So I am still working on other issues I’ve discovered, the installation is simply copy to code folder then run “bin/magento setup:upgrade”, then enable the marketplace plugin under the Magento admin area.
One of the reasons the products may not show is the product filter section in the seller code, I will make more updates in the next week.

From: Godwin Asuquo <notifications@github.com>
Reply to: Apptha/Marketplace-M2 <reply@reply.github.com>
Date: Thursday, 19 December 2019 at 20:11
To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
Cc: andrevanzuydam <andrevanzuydam@gmail.com>, Mention <mention@noreply.github.com>
Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23)


I have not been able to fix it. Please can you tell me step by step how to install and setup this extension on magneto2?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Apptha/Marketplace-M2/pull/23?email_source=notifications&email_token=ABOR7HMIKNLXG4KLVCJORL3QZO2MHA5CNFSM4JSJ6YGKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEHKOOGY#issuecomment-567600923>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABOR7HKVT77AYQFDXG4SSUTQZO2MHANCNFSM4JSJ6YGA>.

ankitdbst commented:
Is this extension freely available now?

andrevanzuydam committed:
Fixes for product adding under seller & checkout

farmerstevieb commented:
> @godilite Hey there sorry for the late reply, did you test with the changes I've pushed already, there are still more errors to fix on the source but I am able to add products now but not attributes?

I can add all products/attributes .... but cannot check out (one page checkout) issues. havent got round to finishing it off yet will drop back in if i get further

andrevanzuydam commented:
Hi Stephen

Depends if you pulled my latest branch, check out should be working as there were things missing there, not sure about attributes but I can look into that.

From: Stephen Bacon <notifications@github.com>
Reply to: Apptha/Marketplace-M2 <reply@reply.github.com>
Date: Tuesday, 19 May 2020 at 12:00
To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
Cc: Andre van Zuydam <andrevanzuydam@gmail.com>, Mention <mention@noreply.github.com>
Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23)


@godilite<https://github.com/godilite> Hey there sorry for the late reply, did you test with the changes I've pushed already, there are still more errors to fix on the source but I am able to add products now but not attributes?

I can add all products/attributes .... but cannot check out (one page checkout) issues. havent got round to finishing it off yet will drop back in if i get further

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA>.

farmerstevieb commented:
> Hi Stephen Depends if you pulled my latest branch, check out should be working as there were things missing there, not sure about attributes but I can look into that. From: Stephen Bacon <notifications@github.com> Reply to: Apptha/Marketplace-M2 <reply@reply.github.com> Date: Tuesday, 19 May 2020 at 12:00 To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com> Cc: Andre van Zuydam <andrevanzuydam@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23) @godilite<https://github.com/godilite> Hey there sorry for the late reply, did you test with the changes I've pushed already, there are still more errors to fix on the source but I am able to add products now but not attributes? I can add all products/attributes .... but cannot check out (one page checkout) issues. havent got round to finishing it off yet will drop back in if i get further — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<[#23 (comment)](https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA>.

Hiya,

On the latest Magento 2 version.
Your latest upload, I have new errors.
invalid form key within admin so cannot view sellers/products/etc etc
Now the count() function (php7.2, 7.3) needs updating within the files (I forgot I had fixed this on my local copy before deploying.
Cannot checkout as of yet as I have to re-fix the count() code issues.

Would you like me to update the files I had changed with your branch? and between us see if we can get this working.

andrevanzuydam commented:
Sure, I was just fixing things because I needed a solution, my pull requests don’t seem to be getting in the main branch.  I am still learning a lot about Magento 2 so I end up struggling quite a bit as well.

From: Stephen Bacon <notifications@github.com>
Reply to: Apptha/Marketplace-M2 <reply@reply.github.com>
Date: Wednesday, 20 May 2020 at 23:27
To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
Cc: Andre van Zuydam <andrevanzuydam@gmail.com>, Mention <mention@noreply.github.com>
Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23)


Hi Stephen Depends if you pulled my latest branch, check out should be working as there were things missing there, not sure about attributes but I can look into that. From: Stephen Bacon notifications@github.com<mailto:notifications@github.com> Reply to: Apptha/Marketplace-M2 reply@reply.github.com<mailto:reply@reply.github.com> Date: Tuesday, 19 May 2020 at 12:00 To: Apptha/Marketplace-M2 Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com> Cc: Andre van Zuydam andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com>, Mention mention@noreply.github.com<mailto:mention@noreply.github.com> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21<https://github.com/Apptha/Marketplace-M2/issues/21> (#23<https://github.com/Apptha/Marketplace-M2/pull/23>) @godilite<https://github.com/godilite>https://github.com/godilite Hey there sorry for the late reply, did you test with the changes I've pushed already, there are still more errors to fix on the source but I am able to add products now but not attributes? I can add all products/attributes .... but cannot check out (one page checkout) issues. havent got round to finishing it off yet will drop back in if i get further — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<#23 (comment)<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA.

Hiya,

On the latest Magento 2 version.
Your latest upload, I have new errors.
invalid form key within admin so cannot view sellers/products/etc etc
Now the count() function (php7.2, 7.3) needs updating within the files (I forgot I had fixed this on my local copy before deploying.
Cannot checkout as of yet as I have to re-fix the count() code issues.

Would you like me to update the files I had changed with your branch? and between us see if we can get this working.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631736378>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABOR7HN47FRMCILLMRDS7WLRSRDNLANCNFSM4JSJ6YGA>.

farmerstevieb commented:
I am in the same boat to be honest, I need something for a small community project, this fitted the bill, but nothing seems to get updated on any of their products which is a shame as they all tick so many boxes.
I will update the code my side and put a pull request to you branch, you can test/merge/change as needed.

Steve

From: Andre van Zuydam <notifications@github.com>
Sent: 21 May 2020 10:09
To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
Cc: Stephen Bacon <steve@farmit.cymru>; Comment <comment@noreply.github.com>
Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23)

Sure, I was just fixing things because I needed a solution, my pull requests don’t seem to be getting in the main branch. I am still learning a lot about Magento 2 so I end up struggling quite a bit as well.

From: Stephen Bacon <notifications@github.com<mailto:notifications@github.com>>
Reply to: Apptha/Marketplace-M2 <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Wednesday, 20 May 2020 at 23:27
To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com>>
Cc: Andre van Zuydam <andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23)


Hi Stephen Depends if you pulled my latest branch, check out should be working as there were things missing there, not sure about attributes but I can look into that. From: Stephen Bacon notifications@github.com<mailto:notifications@github.com<mailto:notifications@github.com%3cmailto:notifications@github.com>> Reply to: Apptha/Marketplace-M2 reply@reply.github.com<mailto:reply@reply.github.com<mailto:reply@reply.github.com%3cmailto:reply@reply.github.com>> Date: Tuesday, 19 May 2020 at 12:00 To: Apptha/Marketplace-M2 Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com%3cmailto:Marketplace-M2@noreply.github.com>> Cc: Andre van Zuydam andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com>>, Mention mention@noreply.github.com<mailto:mention@noreply.github.com<mailto:mention@noreply.github.com%3cmailto:mention@noreply.github.com>> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21<https://github.com/Apptha/Marketplace-M2/issues/21> (#23<https://github.com/Apptha/Marketplace-M2/pull/23>) @godilite<https://github.com/godilite>https://github.com/godilite Hey there sorry for the late reply, did you test with the changes I've pushed already, there are still more errors to fix on the source but I am able to add products now but not attributes? I can add all products/attributes .... but cannot check out (one page checkout) issues. havent got round to finishing it off yet will drop back in if i get further — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub<#23 (comment)<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032>>, or unsubscribehttps://github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA.

Hiya,

On the latest Magento 2 version.
Your latest upload, I have new errors.
invalid form key within admin so cannot view sellers/products/etc etc
Now the count() function (php7.2, 7.3) needs updating within the files (I forgot I had fixed this on my local copy before deploying.
Cannot checkout as of yet as I have to re-fix the count() code issues.

Would you like me to update the files I had changed with your branch? and between us see if we can get this working.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631736378>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABOR7HN47FRMCILLMRDS7WLRSRDNLANCNFSM4JSJ6YGA>.


—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631975741>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AJCXFHZZUK42UPIXPMEH7NDRSTVRJANCNFSM4JSJ6YGA>.

andrevanzuydam commented:
Hi, where are you located?, would you mind us sharing Skype or Slack to
fast track ?


[image: --]

Andre van Zuydam
[image: https://]about.me/andrevanzuydam
<https://about.me/andrevanzuydam?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>

[image: http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam]
<http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam>

[image: https://www.openhub.net/accounts/75973?ref=Detailed]
<https://www.openhub.net/accounts/75973?ref=Detailed>



On Thu, May 21, 2020 at 11:11 AM Stephen Bacon <notifications@github.com>
wrote:

> I am in the same boat to be honest, I need something for a small community
> project, this fitted the bill, but nothing seems to get updated on any of
> their products which is a shame as they all tick so many boxes.
> I will update the code my side and put a pull request to you branch, you
> can test/merge/change as needed.
>
> Steve
>
> From: Andre van Zuydam <notifications@github.com>
> Sent: 21 May 2020 10:09
> To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
> Cc: Stephen Bacon <steve@farmit.cymru>; Comment <
> comment@noreply.github.com>
> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> #21 (#23)
>
> Sure, I was just fixing things because I needed a solution, my pull
> requests don’t seem to be getting in the main branch. I am still learning a
> lot about Magento 2 so I end up struggling quite a bit as well.
>
> From: Stephen Bacon <notifications@github.com<mailto:
> notifications@github.com>>
> Reply to: Apptha/Marketplace-M2 <reply@reply.github.com<mailto:
> reply@reply.github.com>>
> Date: Wednesday, 20 May 2020 at 23:27
> To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com<mailto:
> Marketplace-M2@noreply.github.com>>
> Cc: Andre van Zuydam <andrevanzuydam@gmail.com<mailto:
> andrevanzuydam@gmail.com>>, Mention <mention@noreply.github.com<mailto:
> mention@noreply.github.com>>
> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> #21 (#23)
>
>
> Hi Stephen Depends if you pulled my latest branch, check out should be
> working as there were things missing there, not sure about attributes but I
> can look into that. From: Stephen Bacon notifications@github.com<mailto:
> notifications@github.com<mailto:notifications@github.com%
> 3cmailto:notifications@github.com>> Reply to: Apptha/Marketplace-M2
> reply@reply.github.com<mailto:reply@reply.github.com<mailto:
> reply@reply.github.com%3cmailto:reply@reply.github.com>> Date: Tuesday,
> 19 May 2020 at 12:00 To: Apptha/Marketplace-M2
> Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com
> <mailto:Marketplace-M2@noreply.github.com%
> 3cmailto:Marketplace-M2@noreply.github.com>> Cc: Andre van Zuydam
> andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com<mailto:
> andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com>>, Mention
> mention@noreply.github.com<mailto:mention@noreply.github.com<mailto:
> mention@noreply.github.com%3cmailto:mention@noreply.github.com>> Subject:
> Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21<
> https://github.com/Apptha/Marketplace-M2/issues/21> (#23<
> https://github.com/Apptha/Marketplace-M2/pull/23>) @godilite<
> https://github.com/godilite>https://github.com/godilite Hey there sorry
> for the late reply, did you test with the changes I've pushed already,
> there are still more errors to fix on the source but I am able to add
> products now but not attributes? I can add all products/attributes .... but
> cannot check out (one page checkout) issues. havent got round to finishing
> it off yet will drop back in if i get further — You are receiving this
> because you were mentioned. Reply to this email directly, view it on
> GitHub<#23 (comment)<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032>>,
> or unsubscribehttps://
> github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA.
>
>
> Hiya,
>
> On the latest Magento 2 version.
> Your latest upload, I have new errors.
> invalid form key within admin so cannot view sellers/products/etc etc
> Now the count() function (php7.2, 7.3) needs updating within the files (I
> forgot I had fixed this on my local copy before deploying.
> Cannot checkout as of yet as I have to re-fix the count() code issues.
>
> Would you like me to update the files I had changed with your branch? and
> between us see if we can get this working.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631736378>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/ABOR7HN47FRMCILLMRDS7WLRSRDNLANCNFSM4JSJ6YGA>.
>
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631975741>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AJCXFHZZUK42UPIXPMEH7NDRSTVRJANCNFSM4JSJ6YGA>.
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631977145>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOR7HNFA63DCYSKAMPBSJTRSTV5RANCNFSM4JSJ6YGA>
> .
>

farmerstevieb commented:
I am in Wales(UK).

Sure Skype probably easier as its always on when im on the PC

Steve
From: Andre van Zuydam <notifications@github.com>
Sent: 21 May 2020 10:21
To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
Cc: Stephen Bacon <steve@farmit.cymru>; Comment <comment@noreply.github.com>
Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21 (#23)

Hi, where are you located?, would you mind us sharing Skype or Slack to
fast track ?


[image: --]

Andre van Zuydam
[image: https://]about.me/andrevanzuydam
<https://about.me/andrevanzuydam?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>

[image: http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam]
<http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam>

[image: https://www.openhub.net/accounts/75973?ref=Detailed]
<https://www.openhub.net/accounts/75973?ref=Detailed>



On Thu, May 21, 2020 at 11:11 AM Stephen Bacon <notifications@github.com<mailto:notifications@github.com>>
wrote:

> I am in the same boat to be honest, I need something for a small community
> project, this fitted the bill, but nothing seems to get updated on any of
> their products which is a shame as they all tick so many boxes.
> I will update the code my side and put a pull request to you branch, you
> can test/merge/change as needed.
>
> Steve
>
> From: Andre van Zuydam <notifications@github.com<mailto:notifications@github.com>>
> Sent: 21 May 2020 10:09
> To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com>>
> Cc: Stephen Bacon <steve@farmit.cymru<mailto:steve@farmit.cymru>>; Comment <
> comment@noreply.github.com<mailto:comment@noreply.github.com>>
> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> #21 (#23)
>
> Sure, I was just fixing things because I needed a solution, my pull
> requests don’t seem to be getting in the main branch. I am still learning a
> lot about Magento 2 so I end up struggling quite a bit as well.
>
> From: Stephen Bacon <notifications@github.com<mailto:
<mailto:notifications@github.com%3cmailto:%0b>> notifications@github.com<mailto:notifications@github.com>>>
> Reply to: Apptha/Marketplace-M2 <reply@reply.github.com<mailto:
<mailto:reply@reply.github.com%3cmailto:%0b>> reply@reply.github.com<mailto:reply@reply.github.com>>>
> Date: Wednesday, 20 May 2020 at 23:27
> To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com<mailto:
<mailto:Marketplace-M2@noreply.github.com%3cmailto:%0b>> Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com>>>
> Cc: Andre van Zuydam <andrevanzuydam@gmail.com<mailto:
<mailto:andrevanzuydam@gmail.com%3cmailto:%0b>> andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com>>>, Mention <mention@noreply.github.com<mailto:
<mailto:mention@noreply.github.com%3cmailto:%0b>> mention@noreply.github.com<mailto:mention@noreply.github.com>>>
> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> #21 (#23)
>
>
> Hi Stephen Depends if you pulled my latest branch, check out should be
> working as there were things missing there, not sure about attributes but I
> can look into that. From: Stephen Bacon notifications@github.com<mailto<mailto:notifications@github.com%3cmailto>:
> notifications@github.com<mailto:notifications@github.com%<mailto:notifications@github.com%3cmailto:notifications@github.com%25>
> 3cmailto:notifications@github.com>> Reply to: Apptha/Marketplace-M2
> reply@reply.github.com<mailto:reply@reply.github.com<mailto<mailto:reply@reply.github.com%3cmailto:reply@reply.github.com%3cmailto>:
> reply@reply.github.com%3cmailto:reply@reply.github.com<mailto:reply@reply.github.com%3cmailto:reply@reply.github.com>>> Date: Tuesday,
> 19 May 2020 at 12:00 To: Apptha/Marketplace-M2
> Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com%3cmailto:Marketplace-M2@noreply.github.com>
> <mailto:Marketplace-M2@noreply.github.com%
<mailto:Marketplace-M2@noreply.github.com%25%0b>> 3cmailto:Marketplace-M2@noreply.github.com>> Cc: Andre van Zuydam
> andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com<mailto<mailto:andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com%3cmailto>:
> andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com>>>, Mention
> mention@noreply.github.com<mailto:mention@noreply.github.com<mailto<mailto:mention@noreply.github.com%3cmailto:mention@noreply.github.com%3cmailto>:
> mention@noreply.github.com%3cmailto:mention@noreply.github.com<mailto:mention@noreply.github.com%3cmailto:mention@noreply.github.com>>> Subject:
> Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21<
> https://github.com/Apptha/Marketplace-M2/issues/21> (#23<
> https://github.com/Apptha/Marketplace-M2/pull/23>) @godilite<
> https://github.com/godilite>https://github.com/godilite<https://github.com/godilite%3ehttps:/github.com/godilite> Hey there sorry
> for the late reply, did you test with the changes I've pushed already,
> there are still more errors to fix on the source but I am able to add
> products now but not attributes? I can add all products/attributes .... but
> cannot check out (one page checkout) issues. havent got round to finishing
> it off yet will drop back in if i get further — You are receiving this
> because you were mentioned. Reply to this email directly, view it on
> GitHub<#23 (comment)<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032>>,
> or unsubscribehttps://
> github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA.
>
>
> Hiya,
>
> On the latest Magento 2 version.
> Your latest upload, I have new errors.
> invalid form key within admin so cannot view sellers/products/etc etc
> Now the count() function (php7.2, 7.3) needs updating within the files (I
> forgot I had fixed this on my local copy before deploying.
> Cannot checkout as of yet as I have to re-fix the count() code issues.
>
> Would you like me to update the files I had changed with your branch? and
> between us see if we can get this working.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631736378>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/ABOR7HN47FRMCILLMRDS7WLRSRDNLANCNFSM4JSJ6YGA>.
>
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631975741>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AJCXFHZZUK42UPIXPMEH7NDRSTVRJANCNFSM4JSJ6YGA>.
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631977145>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOR7HNFA63DCYSKAMPBSJTRSTV5RANCNFSM4JSJ6YGA>
> .
>


—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub<https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631981357>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AJCXFH3MER5BPZVCPEAYCEDRSTW65ANCNFSM4JSJ6YGA>.

andrevanzuydam commented:
My Skype is avanzuydam and we're in the same time zone - I'm in South Africa


[image: --]

Andre van Zuydam
[image: https://]about.me/andrevanzuydam
<https://about.me/andrevanzuydam?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>

[image: http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam]
<http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam>

[image: https://www.openhub.net/accounts/75973?ref=Detailed]
<https://www.openhub.net/accounts/75973?ref=Detailed>



On Thu, May 21, 2020 at 11:46 AM Stephen Bacon <notifications@github.com>
wrote:

> I am in Wales(UK).
>
> Sure Skype probably easier as its always on when im on the PC
>
> Steve
> From: Andre van Zuydam <notifications@github.com>
> Sent: 21 May 2020 10:21
> To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com>
> Cc: Stephen Bacon <steve@farmit.cymru>; Comment <
> comment@noreply.github.com>
> Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> #21 (#23)
>
> Hi, where are you located?, would you mind us sharing Skype or Slack to
> fast track ?
>
>
> [image: --]
>
> Andre van Zuydam
> [image: https://]about.me/andrevanzuydam
> <
> https://about.me/andrevanzuydam?promo=email_sig&utm_source=email_sig&utm_medium=email_sig&utm_campaign=external_links>
>
>
> [image: http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam]
> <http://www.daniweb.com/profiles/1108517/Andre-van-Zuydam>
>
> [image: https://www.openhub.net/accounts/75973?ref=Detailed]
> <https://www.openhub.net/accounts/75973?ref=Detailed>
>
>
>
> On Thu, May 21, 2020 at 11:11 AM Stephen Bacon <notifications@github.com
> <mailto:notifications@github.com>>
> wrote:
>
> > I am in the same boat to be honest, I need something for a small
> community
> > project, this fitted the bill, but nothing seems to get updated on any
> of
> > their products which is a shame as they all tick so many boxes.
> > I will update the code my side and put a pull request to you branch, you
> > can test/merge/change as needed.
> >
> > Steve
> >
> > From: Andre van Zuydam <notifications@github.com<mailto:
> notifications@github.com>>
> > Sent: 21 May 2020 10:09
> > To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com<mailto:
> Marketplace-M2@noreply.github.com>>
> > Cc: Stephen Bacon <steve@farmit.cymru<mailto:steve@farmit.cymru>>;
> Comment <
> > comment@noreply.github.com<mailto:comment@noreply.github.com>>
> > Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> > #21 (#23)
> >
> > Sure, I was just fixing things because I needed a solution, my pull
> > requests don’t seem to be getting in the main branch. I am still
> learning a
> > lot about Magento 2 so I end up struggling quite a bit as well.
> >
> > From: Stephen Bacon <notifications@github.com<mailto:
> <mailto:notifications@github.com%3cmailto:%0b>> notifications@github.com
> <mailto:notifications@github.com>>>
> > Reply to: Apptha/Marketplace-M2 <reply@reply.github.com<mailto:
> <mailto:reply@reply.github.com%3cmailto:%0b>> reply@reply.github.com
> <mailto:reply@reply.github.com>>>
> > Date: Wednesday, 20 May 2020 at 23:27
> > To: Apptha/Marketplace-M2 <Marketplace-M2@noreply.github.com<mailto:
> <mailto:Marketplace-M2@noreply.github.com%3cmailto:%0b>>
> Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com>>>
>
> > Cc: Andre van Zuydam <andrevanzuydam@gmail.com<mailto:
> <mailto:andrevanzuydam@gmail.com%3cmailto:%0b>> andrevanzuydam@gmail.com
> <mailto:andrevanzuydam@gmail.com>>>, Mention <mention@noreply.github.com
> <mailto:
> <mailto:mention@noreply.github.com%3cmailto:%0b>>
> mention@noreply.github.com<mailto:mention@noreply.github.com>>>
> > Subject: Re: [Apptha/Marketplace-M2] Fix for: add product showing error
> > #21 (#23)
> >
> >
> > Hi Stephen Depends if you pulled my latest branch, check out should be
> > working as there were things missing there, not sure about attributes
> but I
> > can look into that. From: Stephen Bacon notifications@github.com
> <mailto<mailto:notifications@github.com%3cmailto>:
> > notifications@github.com<mailto:notifications@github.com%<mailto:
> notifications@github.com%3cmailto:notifications@github.com%25>
> > 3cmailto:notifications@github.com>> Reply to: Apptha/Marketplace-M2
> > reply@reply.github.com<mailto:reply@reply.github.com<mailto<mailto:
> reply@reply.github.com%3cmailto:reply@reply.github.com%3cmailto>:
> > reply@reply.github.com%3cmailto:reply@reply.github.com<mailto:
> reply@reply.github.com%3cmailto:reply@reply.github.com>>> Date: Tuesday,
> > 19 May 2020 at 12:00 To: Apptha/Marketplace-M2
> > Marketplace-M2@noreply.github.com<mailto:
> Marketplace-M2@noreply.github.com<mailto:Marketplace-M2@noreply.github.com
> %3cmailto:Marketplace-M2@noreply.github.com>
> > <mailto:Marketplace-M2@noreply.github.com%
> <mailto:Marketplace-M2@noreply.github.com%25%0b>>
> 3cmailto:Marketplace-M2@noreply.github.com>> Cc: Andre van Zuydam
> > andrevanzuydam@gmail.com<mailto:andrevanzuydam@gmail.com<mailto<mailto:
> andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com%3cmailto>:
> > andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com<mailto:
> andrevanzuydam@gmail.com%3cmailto:andrevanzuydam@gmail.com>>>, Mention
> > mention@noreply.github.com<mailto:mention@noreply.github.com
> <mailto<mailto:mention@noreply.github.com%
> 3cmailto:mention@noreply.github.com%3cmailto>:
> > mention@noreply.github.com%3cmailto:mention@noreply.github.com<mailto:
> mention@noreply.github.com%3cmailto:mention@noreply.github.com>>>
> Subject:
> > Re: [Apptha/Marketplace-M2] Fix for: add product showing error #21<
> > https://github.com/Apptha/Marketplace-M2/issues/21> (#23<
> > https://github.com/Apptha/Marketplace-M2/pull/23>) @godilite<
> > https://github.com/godilite>https://github.com/godilite<
> https://github.com/godilite%3ehttps:/github.com/godilite> Hey there sorry
> > for the late reply, did you test with the changes I've pushed already,
> > there are still more errors to fix on the source but I am able to add
> > products now but not attributes? I can add all products/attributes ....
> but
> > cannot check out (one page checkout) issues. havent got round to
> finishing
> > it off yet will drop back in if i get further — You are receiving this
> > because you were mentioned. Reply to this email directly, view it on
> > GitHub<#23 (comment)<
> > https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-630719032>>,
>
> > or unsubscribehttps://
> >
> github.com/notifications/unsubscribe-auth/ABOR7HPHE2BMJJDICBIZJTDRSJKEPANCNFSM4JSJ6YGA.
>
> >
> >
> > Hiya,
> >
> > On the latest Magento 2 version.
> > Your latest upload, I have new errors.
> > invalid form key within admin so cannot view sellers/products/etc etc
> > Now the count() function (php7.2, 7.3) needs updating within the files
> (I
> > forgot I had fixed this on my local copy before deploying.
> > Cannot checkout as of yet as I have to re-fix the count() code issues.
> >
> > Would you like me to update the files I had changed with your branch?
> and
> > between us see if we can get this working.
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub<
> > https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631736378>,
>
> > or unsubscribe<
> >
> https://github.com/notifications/unsubscribe-auth/ABOR7HN47FRMCILLMRDS7WLRSRDNLANCNFSM4JSJ6YGA>.
>
> >
> >
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub<
> > https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631975741>,
>
> > or unsubscribe<
> >
> https://github.com/notifications/unsubscribe-auth/AJCXFHZZUK42UPIXPMEH7NDRSTVRJANCNFSM4JSJ6YGA>.
>
> >
> >
> > —
> > You are receiving this because you were mentioned.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631977145>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ABOR7HNFA63DCYSKAMPBSJTRSTV5RANCNFSM4JSJ6YGA>
>
> > .
> >
>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub<
> https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631981357>,
> or unsubscribe<
> https://github.com/notifications/unsubscribe-auth/AJCXFH3MER5BPZVCPEAYCEDRSTW65ANCNFSM4JSJ6YGA>.
>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Apptha/Marketplace-M2/pull/23#issuecomment-631992705>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABOR7HKVIM7Y4TZEBC3IFBLRSTZ7NANCNFSM4JSJ6YGA>
> .
>
